src/ast.cpp

```diff
@@ -32,7 +33,68 @@ void CreateTableCommand::execute(Database& database)
 
 void InsertIntoCommand::execute(Database& database)
 {
-  std::cout << *this << std::endl;
+  using namespace white::davisbase::sdl;
+  using common::NullValue;
+  using std::holds_alternative;
+
+  auto table_opt = database.getTable(table_name);
+  if (table_opt == std::nullopt) {
+    throw std::runtime_error("Table doesn't exist");
+  }
+
+
+  std::map<std::string, common::ColumnDefinition> columns_map;
+  std::vector<common::ColumnDefinition> column_defs =
+    table_opt.columnDefinitions();
+
+  for (int i = 0; i < column_defs.size(); i++) {
+    columns_map[column_defs[i].name] = column_defs[i];
+  }
+
+  if (column_names.size() > 0) {
+    for (int i = 0; i < column_names.size(); i++) {
+
+      if (columns_map[column_names[i]] != std::nullopt) {
```

Comment:
```
/home/travis/build/ozars/white-davisbase/src/ast.cpp:57:40: error: no match for ‘operator!=’ (operand types are ‘std::map<std::__cxx11::basic_string<char>, white::davisbase::common::ColumnDefinition>::mapped_type’ {aka ‘white::davisbase::common::ColumnDefinition’} and ‘const std::nullopt_t’)
```

`nullopt` can be used only with comparing with `optional` values, it's not a generic null. Please use `map.count(key)` or `map.find(key) != map.end()` to check if an element exists (if `operator[]` is used it will insert an element first if it doesn't exist).